text.cabal

```diff
@@ -161,10 +160,7 @@ library
     ghc-options: -Werror
     cpp-options: -DASSERTS
 
-  if flag(integer-simple)
-    cpp-options: -DINTEGER_SIMPLE
-    build-depends: integer-simple >= 0.1 && < 0.5
-  else
+  if flag(integer-gmp)
     cpp-options: -DINTEGER_GMP
```

Comment:
We can actually simplify this even more; i.e. we can get rid of `-DINTEGER_GMP` as it's kinda redundant; use

```c
#if defined(MIN_VERSION_integer_gmp)
/* assume GMP is available */
#elif defined(MIN_VERSION_integer_simple)
/* assume integer-simple is used */
#else
# error unsupported configuration
#endif
```

instead  to detect which one of those packages is in scope (and assume that

Reply:
Good idea - changed in https://github.com/haskell/text/pull/264/commits/34736352382a16bb20a6083b06cb48aec8f9aaf8